partner_event/views/res_partner_view.xml

```diff
@@ -1,32 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data>
+<!-- © 2016 Tecnativa S.L. - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
 
 <record id="act_partner_registration" model="ir.actions.act_window">
     <field name="res_model">event.registration</field>
     <field name="view_type">form</field>
     <field name="name">Registrations</field>
     <field name="view_mode">tree,form,calendar,graph</field>
-    <field name="context">{'search_default_partner_id': active_id, 'default_partner_id': active_id}</field>
+    <field name="domain">[("partner_id", "child_of", active_ids)]</field>
+    <field name="context">{'default_partner_id': active_id, 'search_default_expected': True}</field>
```

Comment:
No, they are different things. I fear that the filter activated with this context key has been replaced with a specific domain like `('id', 'in', ids)`. I prefer this form as it allows to refresh and it performs better.